qualtricssurvey/views.py

```diff
@@ -104,6 +106,11 @@ def provide_context(self, context=None):
             'show_simulation_exists_string': show_simulation_exists_string,
             'show_meta_information_string': show_meta_information_string,
             'message': self.message,
+            'survey_completed': param_survey_completed,
+            'anon_user_id_string': anon_user_id_string,
+            'earned_score': self.score.raw_earned,
```

Comment:
@bsande6 It appears that when creating a new `qualtricssurvey` in the CMS that `self.score` here is NoneType. Can you check for this and return 0 if `self.score` is NoneType?

```
Traceback (most recent call last):
  File "/edx/app/edxapp/edx-platform/cms/djangoapps/contentstore/views/preview.py", line 327, in get_preview_fragment
    fragment = module.render(preview_view, context)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.5/site-packages/xblock/core.py", line 202, in render
    return self.runtime.render(self, view, context)
  File "/edx/app/edxapp/edx-platform/common/lib/xmodule/xmodule/x_module.py", line 1979, in render
    return self.__getattr__('render')(block, view_name, context)
  File "/edx/app/edxapp/edx-platform/common/lib/xmodule/xmodule/x_module.py", line 1423, in render
    return super(MetricsMixin, self).render(block, view_name, context=context)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.5/site-packages/xblock/runtime.py", line 829, in render
    frag = view_fn(context)
  File "/edx/src/xblocks/xblock-qualtrics-survey/qualtricssurvey/mixins/fragment.py", line 64, in student_view
    context = self.provide_context(context)
  File "/edx/src/xblocks/xblock-qualtrics-survey/qualtricssurvey/views.py", line 111, in provide_context
    'earned_score': self.score.raw_earned,
AttributeError: 'NoneType' object has no attribute 'raw_earned'
```

Reply:
I updated this with https://github.com/CUCWD/xblock-qualtrics-survey/pull/6/commits/c1f00430b50ab3727a50fd333455c9500bc92156

Reply:
Handled update with https://github.com/CUCWD/xblock-qualtrics-survey/commit/6c1017df7b3e50f120bc7829cc32abf84adf05a0